### PR TITLE
prettier support for new files

### PR DESCRIPTION
**Pull request by bastianwegge**
added prettier, husky, lint-staged to dev-dependencies and enabled precommit with prettier + .prettierrc

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "printWidth": 80,
+  "parser": "babylon",
+  "singleQuote": true
```
Comment (mairh) [inline .prettierrc]: Why don't we go with the default settings? I don't have any strong opinion on using single vs double quotes, but I definitely prefer using the default setting from a third party lib as long as it is safe.

Comment (bastianwegge) [reply .prettierrc]: printWidth + parser are default and just added to show the settings, doubleQuote everywhere is the default, and as far as I can remember single quote is airbnb (so our eslint) default

**Comment by vlasenko** [comment]
Should I merge it, everyone agree?

**Comment by mairh** [comment]
Yes

**Comment by vlasenko** [comment]
Okay, lets rock

**Pull request merged**